File: PlacementicsBackend/src/main/java/com/example/placementicsbackend/services/mentormeentee/interfaces/IEmailService.java
package com.example.placementicsbackend.services.mentormeentee.interfaces;

import java.time.LocalDateTime;

public interface IEmailService {

    boolean sendMeetingInvite(
            String recipientEmail,
            String recipientName,
            String meetingTitle,
            String meetingDescription,
            LocalDateTime startTime,
            LocalDateTime endTime,
            String meetLink,
            String otherAttendeeEmail);
}

File: PlacementicsBackend/src/main/java/com/example/placementicsbackend/services/MentorMenteeService/Interface/IEmailService.cs
namespace PlacementCellBackend.Services.MentorMenteeService.Interface;

public interface IEmailService
{
    Task<bool> SendMeetingInviteAsync(
        string recipientEmail,
        string recipientName,
        string meetingTitle,
        string meetingDescription,
        DateTime startTime,
        DateTime endTime,
        string meetLink,
        string otherAttendeeEmail);
}
